client/src/pages/Home/Footer/Footer.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import "./Footer.css";
+import { FacebookOption, Twitter, Instagram } from "grommet-icons";
+
+export function Footer() {
+	const [isShown, setIsShown] = useState(false);
+	return (
+		<div id="footer">
+			{/* Section11 */}
+			<div className="home_section11_parent">
+				<div className="home_section11_child1">
+					<img
+						className="footer-logo"
+						src="https://i.postimg.cc/VkdMYP4S/Logo.jpg"
+						alt="logo"
+					/>
+				</div>
+				<div className="home_section11_child2">
+					<div>
+						<p>Do want to be part of us?</p>
```

Comment:
please add `you` in there: `Do want to be part of us?` :)

Reply:
No, we don't need the following file.

Footer.js Footer.css I need to delete the folder.